pkg/core/interop/contract/call.go

```diff
@@ -69,6 +69,26 @@ func Call(ic *interop.Context) error {
 		return fmt.Errorf("method not found: %s/%d", method, len(args))
 	}
 	hasReturn := md.ReturnType != smartcontract.VoidType
+
+	if ic.Chain.GetConfig().Ledger.SaveInvocations {
+		arr := stackitem.NewArray(args)
+		arrCount := len(args)
+		valid := true
+		argBytes := []byte{}
```

Comment:
Do you need this to be initialized to zero-length slice? Also, stylistically I'd prefer
```
var (
)
```
block here since you declare a lot of things.